### ISSUE TITLE
LND Auto Unlock script fails: certificate is valid for 127.0.0.1 ..... not 0.0.0.0

### ISSUE THREAD

**Issue by oitdmser** [Stadicus/RaspiBolt#448]
Implemented the update LND auto unlock script that is supposed to work with LND 0.5. When I run the script it fails to unlock the wallet. The script's debug.log says:

`{"error":"all SubConns are in TransientFailure, latest connection error: connection error: desc = \"transport: authentication handshake failed: x509: certificate is valid for 127.0.0.1, ::1, 10.1.10.20, fe80::1a03:73ff:fed1:9ca1, not 0.0.0.0\"","code":14}`

I checked that lnd.conf has restlisten=127.0.0.1:8080.

**Comment by oitdmser** [comment]
Just ran across this similar issue (#352) [https://github.com/Stadicus/RaspiBolt/issues/352](url) The solution there was to add the ip address using `tlsextraip=X.X.X.X` in `lnd.conf`, delete tls.cert and tls.key and restart the node to generate new tls info. I plan to try this soon and will report back

**Comment by oitdmser** [comment]
My LND node is run with REST and gRPC interfaces open to the LAN as well as localhost. My original lnd.conf simply had `rpclisten=0.0.0.0:10009` and `restlisten=0.0.0.0:8080`. 

Step 1.
I tried adding `tlsextraip=0.0.0.0` deleting *.tls in ~/.lnd and restarting, no workie. In the linked issue above some found it worked by deleting *.tls and restarting a second time, so I tried a second. No workie.

Step 2.
The [guide](https://stadicus.github.io/RaspiBolt/raspibolt_6A_auto-unlock.html) says the unlock script just uses REST, so I deleted `tlsextraip=0.0.0.0` and instead of `restlisten=0.0.0.0:8080` I used two lines: `restlisten=my.ip.add.ress:8080` and `restlisten=127.0.0.1:8080` Deleted *.tls and restarted lnd. Still no workie.

Step 3.
Just shooting in the dark now. I now changed my `rpclisten=` to be `rpclisten=my.ip.add.ress:8080` and `rpclisten=127.0.0.1:8080`. Deleted *.tls and restarted and now the unlock script works.

Posting the solution in case it helps someone else.
